### PR TITLE
doc/readme : make clear that the links are just examples

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Description, Motivation and Context

Just ensure that users might not be misleading when they are trying to understand the replicated org solutions and does not know that those links are only preflight examples and has no relation with kURL or Kots. 

## Checklist

- [ ] New and existing tests pass locally with the changes.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] The commit message(s) are informative and highlight any breaking changes

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ curl https://krew.sh/preflight | bash
  and run:
  
 ```
+# It is ONLY an example. Do **not** use to validate cluster installs.
```
Comment (camilamacedo86) [inline README.md]:
```suggestion
# It is ONLY an example. Do **not** use to validate cluster state.
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ curl https://krew.sh/preflight | bash
  and run:
  
 ```
+# It is ONLY an example. Do **not** use to validate cluster state.
```
Comment (camilamacedo86) [inline README.md:18]:
```suggestion
# It is ONLY an example. Do **not** use to validate real scenarios.
```

File: README.md [hunk 3]
```diff
@@ -32,8 +33,10 @@ curl https://krew.sh/support-bundle | bash
  and run:
  
 ```
+# It is ONLY an example. Do **not** use to validate applications.
```
Comment (camilamacedo86) [inline README.md:36]:
```suggestion
# It is ONLY an example. Do **not** use to validate real scenarios.
```

File: README.md [hunk 4]
```diff
@@ -15,6 +15,7 @@ curl https://krew.sh/preflight | bash
  and run:
  
 ```
+# It is ONLY an example. Do **not** use to validate real scenarios.
```
Comment (xavpaice) [inline README.md]: I'd like to move this out of the command example and have it directly above.  e.g.:

"and run:" becomes

"and run, where  https://preflight.replicated.com provides an example preflight spec:"

we should also add a note to "To run a sample preflight check from a sample application,":

"To run a sample preflight check from a sample application (NOTE: this is an example. Do **not** use to validate real scenarios.):"

**Review by xavpaice — COMMENTED** [review]
Great idea - as noted inline, I'd prefer to have the example note outside of copyable text if possible.

It's a nit, not blocking on that.

File: README.md (hunk 4 above)
Comment (camilamacedo86) [reply README.md]: changed let me wdyt?

**Review by xavpaice — APPROVED** [review]
This is great, thanks!

**Pull request merged**